RuntimePerformanceTests/Package.resolved

```diff
@@ -1,22 +1,21 @@
 {
-  "originHash" : "429b40215857f8815df868444cc297cc285e2ed32bfe603593f661571d61634d",
+  "originHash" : "c904a6a501a9d1f10fa2dd02f91cfba2ca658033d434e746e671ebf936476691",
   "pins" : [
     {
       "identity" : "hdrhistogram-swift",
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/HdrHistogram/hdrhistogram-swift",
       "state" : {
-        "revision" : "a69fa24d7b70421870cafa86340ece900489e17e",
-        "version" : "0.1.2"
+        "revision" : "93a1618c8aa20f6a521a9da656a3e0591889e9dc",
+        "version" : "0.1.3"
       }
     },
     {
       "identity" : "package-benchmark",
       "kind" : "remoteSourceControl",
-      "location" : "https://github.com/ordo-one/package-benchmark",
+      "location" : "https://github.com/loonatick-src/package-benchmark",
       "state" : {
-        "revision" : "51fffb6bc58bcbaca9c1954e6e52b56f3c9c2d54",
-        "version" : "1.27.4"
+        "revision" : "94f97d3c8e766869c641129f73bed83feb422ae5"
```

Comment:
Looks like the upstream has been merged, just dropping this as a reminder comment to change back from the fork before merge